### DISCUSSION TITLE
"Search Failed: there is .... " too chatty!

### DISCUSSION THREAD

**Discussion by amno1** [Fuco1/smartparens#1190] [Q&A]
Can we turn off that madness somehow???

If anything gets me to abandon Smartparens package, it will probably be this annoying chatter.

Can it be so that Smartparens does not ignore commented code? When I do check with "check-parens" it throws me into commented code. Can it be that Smartparens is also going through commented code? If case, can we disable the behaviour?

**Comment by Fuco1** [comment]
Yes there is a defcustom for this.  Check `sp-message-width`

**Comment by amno1** [comment]
Aha! Thanks you very much. I will look for that option.



Skickat från min Samsung Galaxy-smartphone.


-------- Originalmeddelande --------
Från: Matus Goljer <notifications@github.com>
Datum: 2019-02-06 19:54 (GMT+01:00)
Till: Fuco1/smartparens <smartparens@noreply.github.com>
Kopia: amno1 <arthur.miller@live.com>, Author <author@noreply.github.com>
Ämne: Re: [Fuco1/smartparens] "Search Failed: there is .... " too chatty! (#964)


Yes there is a defcustom for this. Check sp-message-width
